language/move-binary-format/src/file_format.rs

```diff
@@ -651,6 +651,10 @@ impl AbilitySet {
             | (Ability::Key as u8),
     );
 
+    pub fn singleton(ability: Ability) -> Self {
```

Comment:
FWIW, this is just `AbilitySet::EMPTY | ability`, which I do quite often. But this API isn't bad :)